### ISSUE TITLE
Default binding broken

### ISSUE THREAD

**Issue by awlayton** [rcaloras/bashhub-client#77]
After install, pressing ctrl-b results in the below error:

```shell
 ^Kbh -i

Command '
         bh' not found, did you mean:

  command 'bh' from deb buthead

Try: sudo apt install <deb name>
```

OS: WSL Ubuntu 18.04.4
shell version:

```shell
$ bash --version
GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

**Comment by rcaloras** [comment]
Hey @awlayton, thanks for the feedback. Just to confirm, can you execute `bh` or `bashhub` itself on the command line? Or this is specific to the binding? The error looks like it can't find `bh`.

**Comment by awlayton** [comment]
Yes the error is specific to the binding. I can run both `bh` and `bashhub` myself just fine. It looks like the binding is trying to run "<vertical line feed>bh -i" rather than bh -i

**Comment by awlayton** [comment]
I was able to make it work for me by changing the alias command to `bind '"\C-b":"\C-ubh -i\n"`

Not sure what pressing Ctrl-K is supposed to do but in my setup that just types out `^K`

**Comment by rcaloras** [comment]
Thanks for reporting @awlayton. Fixed in the next release on 1.2.1-dev branch. 👍
